ai_tools/elasticgraph-mcp-server/src/elasticgraph_mcp/server.py

```diff
@@ -0,0 +1,242 @@
+                    "details": "Could not fetch documentation from remote server",
+                },
+            )
+
+        return {"contents": [{"uri": "docs://api", "mimeType": "text/plain", "text": docs}]}
+    except McpError:
+        raise
+    except Exception as e:
+        raise create_command_error(
+            "Failed to fetch API documentation",
+            error=e,
+            data={
+                "hint": "Check your internet connection and try again",
+                "details": "An unexpected error occurred while fetching documentation",
+            },
+        ) from e
+
+
+@mcp.resource("usage://commands")
+def get_common_commands() -> ResponseDict:
+    """
+    Lists common ElasticGraph commands grouped by category.
+    """
+    try:
+        commands = [
+            {
+                "command": "gem exec elasticgraph new PROJECT_NAME",
+                "description": "Create a new ElasticGraph project in a new PROJECT_NAME directory",
+                "example": "gem exec elasticgraph new my_project",
+            },
+            {
+                "command": "bundle exec rake boot_locally",
+                "description": "Start ElasticGraph for development, requires Docker",
+                "example": "cd my_project && bundle exec rake boot_locally",
+            },
+            {
+                "command": "bundle exec rake schema_artifacts:dump",
+                "description": "Validate and dump the Ruby schema definition into schema.graphql",
+                "example": "cd my_project && bundle exec rake schema_artifacts:dump",
+            },
+            {
+                "command": "bundle exec rake -T",
+                "description": "List available rake tasks with descriptions",
+                "example": "cd my_project && bundle exec rake -T",
+            },
+        ]
+
+        return {
+            "contents": [
+                {"uri": "usage://commands", "mimeType": "application/json", "text": json.dumps({"commands": commands})}
+            ]
+        }
+    except Exception as e:
+        raise create_command_error(
+            "Failed to generate commands list",
+            error=e,
+            data={
+                "hint": "This is an internal error, please report it",
+                "details": "Error occurred while generating commands list",
+            },
+        ) from e
+
+
+@mcp.resource("docs://links")
+def get_documentation_links() -> ResponseDict:
+    """
+    Get links to ElasticGraph documentation and resources.
+    """
+    try:
+        links = {
+            "homepage": {
+                "url": "https://block.github.io/elasticgraph/",
+                "description": "ElasticGraph official website",
+            },
+            "query_api": {
+                "url": "https://block.github.io/elasticgraph/query-api/",
+                "description": "ElasticGraph Query API",
+            },
+            "guide": {
```

Comment:
```suggestion
            "quickstart": {
```

We have a growing `/guides` section of the site so seemed weird to call this `guide`.